src/containers/add-resources/AddResources.tsx

```diff
@@ -60,12 +61,14 @@ const AddResources = <T extends CourseResources | Question>({
 
   const onResponseError = useCallback(
     (error: ErrorResponse) => {
-      setAlert({
-        severity: snackbarVariants.error,
-        message: error ? `errors.${error.code}` : ''
-      })
+      dispatch(
+        openAlert({
+          severity: snackbarVariants.error,
+          message: error ? `errors.${error.code}` : ''
```

Comment:
Maybe it's better to set up some default error instead of just empty string? I saw below we have UNKNOWN_ERROR constant
I saw some alert with just icon and it'll look a little bit confusing, how do you think?
And if you agree, please don't forget to change it in all occurrences🥹

Reply:
I agree with you. I made function like this ```
const getErrorKey = (error) =>
  `errors.${error && error.code ? error.code : 'UNKNOWN_ERROR'}`
```
 to return error translation key. Used it in all the occurrences.